testing/vault/vault.post-deinstall

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# Delete vault group and user
+delgroup vault 2> /dev/null
+deluser vault 2> /dev/null
```

Comment:
I understand, will take it out. (Actually copied from consul package, might need to fix that there as well)